tap_mssql/sync_strategies/common.py

```diff
@@ -141,6 +144,11 @@ def row_to_singer_record(catalog_entry, version, row, columns, time_extracted, c
             row_to_persist += (boolean_representation,)
         elif isinstance(elem, uuid.UUID):
             row_to_persist += (str(elem),)
+        elif property_format == 'singer.decimal':
+            if elem is None:
+                row_to_persist += (elem,)
+            else:
+                row_to_persist += (str(elem),)

```

Comment:
`singer.decimal` `RECORD` message processing. The numeric values are set as `str()` with the exception of `None` where we do not want records to come through with `'None'` as a value